assets/utils.tsx

```diff
@@ -513,6 +513,8 @@ export function errorHandler(error: {errorData: any} | Response, dispatch?: any,
     if ('errorData' in error) {
         if (setError) {
             dispatch(setError(error.errorData));
+            const errorMessage = Object.values(error.errorData).join(', ');
```

Comment:
Why are you displaying a notification for the validation error? This isn't the way validation currently works. The error should be displayed in the form, directly underneath the input that caused the error.

Have a look at the Company form, if you try and create 2 companies with the same name

Reply:
yep, I checked. It seems to me that this is also core work. not related to async